requirements.txt

```diff
@@ -1,5 +1,6 @@
 browserid
```

Comment:
I think this should be `bidpom` now, though updating it can be a separate commit, perhaps even a separate pull.